release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -739,6 +770,10 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, an old version of the Kubernetes client library was present in the `baremetal-runtimecfg` project. When a Virtual IP (VIP) failed, the client connections might not be closed in a timely manner. This could result in long delays for monitor containers that rely on talking to the API. This fix updates the client library. Now, connections are closed as expected on VIP failovers and the monitor containers do not hang for an excessive period of time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1995021[*BZ#1995021*])
 
+* Before this update, the Machine Config Operator (MCO) stored pending configuration changes to disk before it applied them to {op-system-first}. If a power loss interrupted the MCO from applying the configuration, it treated the configuration as applied and did not validate the changes. If this configuration contained invalid changes, applying them failed. With this update, the MCO saves a configuration to disk only after being applied. This way, if the power is lost while the MCO is applying the configuration, it reapplies the configuration after it restarts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916169[BZ#1916169])
```

Comment:
These two aren't bolded, and the one below (that was already there) has a quote (") in error